osu.Game/Screens/SelectV2/UserLinkContainer.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Localisation;
+using osu.Game.Online;
+using osu.Game.Online.Chat;
+using osu.Game.Overlays;
+using osu.Game.Users;
+
+namespace osu.Game.Screens.SelectV2
+{
+    public partial class UserLinkContainer : OsuHoverContainer, IHasContextMenu
+    {
+        private readonly TruncatingSpriteText text;
+
+        public float MaxWidth
+        {
+            set => text.MaxWidth = value;
+        }
```

Comment:
unused?

Reply:
good find, it's supposed to be used in `MetadataDisplay`.